testsuite/tests/kuadrant/authorino/identity/rhsso/test_jwt_ttl.py

```diff
@@ -0,0 +1,66 @@
+"""
+Test for JWT TTL (time to live), see
+https://github.com/Kuadrant/authorino/blob/main/docs/features.md#jwt-verification-authenticationjwt
+"""
+
+from time import sleep
+
+import pytest
+
+pytestmark = [pytest.mark.authorino]
+
+
+def test_jwt_ttl(client, auth, rhsso, create_jwt, jwt_ttl):
+    """
+    Note that similar test exists also in test_dinosaur.py. If modifying this test update the test there too.
+
+    Test:
+        - send request using user with valid middle name
+        - assert that response status code is 200
+        - delete the signing RS256 key from JWKS - the one associated with the JWT for user with valid middle name
+        - assert that response status code is still 200 due to deleted key being cached by Authorino
+        - sleep to allow for Authorino to update the cache as part of OIDC discovery
+        - assert that response status is 401 after cache update
+        - create a new signing RS256 key in JWKS to be used for generating new JWTs
+        - generate a new JWT
+        - assert that response status code is 200 if using new JWT
+        - assert that response status is still 401 for old JWT - ie old JWT does not work with new signing RS256 key
+
+    """
+    response = client.get("/anything/dinosaurs_mgmt/v1/dinosaurs", auth=auth)
```

Comment:
you could simplify the URL to just /get

Reply:
Good catch! I am sorry, copy paste error. Will force push once we agree on how to address your other comments